code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -178,3 +174,21 @@
 	new /obj/item/construction/plumbing(src)
 	new	/obj/item/plunger(src)
 	new	/obj/item/plunger(src)
+
+/obj/structure/closet/secure_closet/chemical/plumbing
+	name = "chemical plumbing closet"
+	desc = "For all your plumbing needs (toilet plumbing not included)."
+	
+/obj/structure/closet/secure_closet/chemical/plumbing/PopulateContents()
+	new /obj/item/stack/ducts/fifty(src)
+	new /obj/item/stack/ducts/fifty(src)
+	new /obj/item/stack/ducts/fifty(src)
+	new /obj/item/stack/ducts/fifty(src)
+	new /obj/item/construction/plumbing(src)
+	new /obj/item/construction/plumbing(src)
+	new /obj/item/rcd_ammo(src)
+	new /obj/item/rcd_ammo(src)
+	new /obj/item/rcd_ammo(src)
+	new /obj/item/rcd_ammo(src)
+	new /obj/item/rcd_ammo(src)
+	new /obj/item/rcd_ammo(src)
```

Comment:
Same here, you can probably get away with doing 4 of them since each one has so many charges.

Reply:
And done.